docs/quickstart.md

```diff
@@ -129,6 +105,14 @@ To download the results simply call the `get_results` method on the order:
 results = order.get_results()
 ```
 
+## Credits and Billing
+
+When you first create an account on Rapidata, you will receive 100 free credits. Each credit give you 10 responses. After you have used up your free credits, you may purchase more [here](https://app.rapidata.ai/pricing).
+
```

Comment:
don't you think it's a bit confusing to have links behind "dashboard" that link to different pages?

Reply:
Idk, but in like 8th grade I had a voluntary html course at school and the teacher called this the _Here Syndrome_ and ranted for a solid 30 minutes about people using stuff like: 

More info can be found [here](https://rapidata.ai)

Instead of 

More info on our [website](https://rapidata.ai)